plane_mcp/tools/work_item_properties.py

```
@@ -73,6 +83,17 @@ def _get_workspace_props_for_type(type_id: str) -> list:
         if not project_id:
             return _get_workspace_props_for_type(work_item_type_id)
 
+        if not work_item_type_id:
+            # project-flat endpoint: all properties in the project, any type
+            try:
+                return client.work_item_properties.list_project(
+                    workspace_slug=workspace_slug,
+                    project_id=project_id,
+                    params=params,
+                )
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)